pkg/indexmanagement/scripts.go

```diff
@@ -35,12 +37,21 @@ from __future__ import print_function
 import json,sys
 r=json.load(sys.stdin)
 alias="${POLICY_MAPPING}-write"
-indices = [index for index in r if r[index]['aliases'][alias]['is_write_index']]
-if len(indices) > 0:
-  print(indices[0])
+try:
+  indices = [index for index in r if r[index]['aliases'][alias]['is_write_index']]
+  if len(indices) > 0:
+    print(indices[0])
+except:
+  e = sys.exc_info()[0]
+  sys.stderr.write("Error trying to determine the 'write' index from '%r': %r" % (r,e))
+  sys.exit(1)
```

Comment:
don't we have the same concern here?

Reply:
This one I don't thinks so.  Would it not mean there is a fundamental error with what the 'alias' request means from the REST api?

Reply:
no harm in coding defensively so we don't have to circle back on this though

Reply:
This particular case I don't see the value in walking the dict to ensure its correct.  We are looking for one specific index and if it fails:
* we get the thing being walked in the error
* we have to rely upon the API at some point being reliable